File: BackEnd/src/main/java/com/unosquare/admin_core/back_end/dto/EmployeeDTO.java
package com.unosquare.admin_core.back_end.dto;

import lombok.Data;

import java.time.LocalDate;

@Data
public class EmployeeDTO {

    private int employeeId;

    private String forename;

    private String surname;

    private String email;

    private int totalHolidays;

    private LocalDate startDate;

    private int countryId;

    private String countryDescription;

    private int employeeRoleId;

    private String employeeRoleDescription;

    private int employeeStatusId;

    private String statusDescription;

    private String password;
}

File: v2/AdminCore.DTOs/Employee/EmployeeDto.cs
using System;

namespace AdminCore.DTOs.Employee
{
  public class EmployeeDto
  {
    public int CountryId { get; set; }

    public string Email { get; set; }

    public int EmployeeId { get; set; }

    public string Forename { get; set; }

    public string Password { get; set; }

    public int EmployeeRoleId { get; set; }

    public DateTime StartDate { get; set; }

    public int EmployeeStatusId { get; set; }

    public string Surname { get; set; }

    public int TotalHolidays { get; set; }
  }
}